oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+stock-logistics-warehouse
```

Comment:
Not sure this one is useful as `MQT_DEP=PIP` is enabled here: https://github.com/OCA/stock-logistics-workflow/blob/14.0/.travis.yml#L30

Reply:
thx @sebalix , I'll remove it